ros/processor/insights_engine_result_consumer.py

```diff
@@ -196,11 +196,11 @@ def process_report(self, host, platform_metadata, reports, utilization_info, per
                 processor_requests_success.labels(
                     reporter=self.reporter, account_number=host['account']
                 ).inc()
-                LOG.info("%s - Refreshed system %s (%s) belonging to account: %s (%s).",
-                         self.prefix, system.inventory_id, system.id, account.account, account.id)
+                LOG.info("%s - Refreshed system %s (%s) belonging to account: %s (%s) and org_id: %s.",
```

Comment:
Along with these log messages, could you also look into code example given here - https://consoledot.pages.redhat.com/docs/dev/platform-documentation/tools/logging.html which explain how to use CloudWatch API to publish logs.
To add filter on request_id/account number/org_id, please check how to extend `ContextualFilter` method.

In my opinion, this will help us while debugging any issue.

Reply:
We can extend `logging.Filter` class to add one or more specific filter methods.
https://docs.python.org/3/library/logging.html#filter-objects for more details.

Reply:
@r14chandra  - As this PR is relevant to `EBS to org_id` feature, created a separate JIRA card to track refactoring change - https://issues.redhat.com/browse/RHIROS-715